README.md

```diff
@@ -1,16 +1,13 @@
 # Mu for Haskell
 
```

Comment:
You can add this to the main README.md, it always looks nice! 😄 
[![Build Status](https://travis-ci.com/higherkindness/mu-haskell.svg?branch=master)](https://travis-ci.com/higherkindness/mu-haskell)

Reply:
I'll add it, how it never shows otherwise xD